modules/openapi-generator/src/main/resources/typescript-axios/package.mustache

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "{{npmName}}",
+  "version": "{{npmVersion}}",
+  "description": "OpenAPI client for {{npmName}}",
+  "author": "OpenAPI-Generator Contributors",
+  "keywords": [
+    "fetch",
+    "typescript",
+    "openapi-client",
+    "openapi-generator",
+    "{{npmName}}"
+  ],
+  "license": "Unlicense",
+  "main": "./dist/index.js",
+  "typings": "./dist/index.d.ts",
+  "scripts" : {
+    "build": "tsc --outDir dist/",
+    "prepublishOnly": "npm run build"
+  },
+  "dependencies": {
```

Comment:
Do we need to add `axios` dependency (peer, dev) somehow?

e.g. https://github.com/sheaivey/react-axios/blob/master/package.json#L45, https://github.com/sheaivey/react-axios/blob/master/package.json#L50

Disclaimer: I'm no expert in TS/JS

Reply:
You are right, sorry I missed that.

Commit is already pushed.